documentation/api/development.md

```diff
@@ -11,17 +11,28 @@ A very simple [docker-compose.yml](../../docker-compose.yml) has been included t
 1. Install the version of Python specified in [pyproject.toml](../../api/pyproject.toml)
    [pyenv](https://github.com/pyenv/pyenv#installation) is one popular option for installing Python,
    or [asdf](https://asdf-vm.com/).
-
-2. After installing and activating the right version of Python, install
+   - If using pyenv run `pyenv local <version>` to ensure that version will be used in subsequent steps
+2. Ensure that `python -V` and `python3 -V` are picking up that version.
+   - If not, run `pyenv init -` and/or restart your shell to ensure it was run automatically
+3. After installing and activating the right version of Python, install
    [poetry](https://python-poetry.org/docs/#installation) and follow the instructions to add poetry to your path if necessary.
 
    ```bash
    curl -sSL https://install.python-poetry.org | python3 -
    ```
 
-3. If you are using an M1 mac, you will need to install postgres as well: `brew install postgresql` (The psycopg2-binary is built from source on M1 macs which requires the postgres executable to be present)
+4. If you are using an M1 mac, you will need to install postgres as well: `brew install postgresql` (The psycopg2-binary is built from source on M1 macs which requires the postgres executable to be present)
```

Comment:
Ah, this was written a while ago, we're not using psycopg2 anymore, and are using version 3 which is _just_ psycopg.

Reply:
Removed.